utils.go

```diff
@@ -15,8 +15,13 @@ import (
 	"strings"
 	"sync"
 	"syscall"
+	"unsafe"
 
 	"golang.org/x/sys/unix"
+
+	"github.com/DataDog/ebpf-manager/elfmap"
```

Comment:
Is there a separate branch that delivers the `elfmap` package ? I was not able to find it here.

Reply:
@r1viollet sorry didn't see your msg, here the elfmap package

Reply:
No problem, if you resume this, we also did a prototype at one point here : https://github.com/Gui774ume/utrace/pull/12/files
I'm  not sure if that can be useful.